content/references/rippled-api/ledger-data-formats/ledger-object-types/negativeunl.md

```diff
@@ -0,0 +1,64 @@
+---
+html: negativeunl.html
+funnel: Build
+doc_type: References
+supercategory: rippled API
+category: Ledger Data Formats
+subcategory: Ledger Object Types
+status: not_enabled
+blurb: List of validators currently believed to be offline.
+---
+# NegativeUNL
+
+_(Requires the [NegativeUNL amendment]() :not_enabled:)_
+
+The `NegativeUNL` object type contains the current status of the [Negative UNL](negative-unl.html), a list of trusted validators currently believed to be offline.
+
+Each ledger version contains **at most one** `NegativeUNL` object. If no validators are currently disabled or scheduled to be disabled, there is no `NegativeUNL` object in the ledger.
+
+## Example {{currentpage.name}} JSON
+
+```json
+{
+  "DisabledValidators": [
+    {
+      "DisabledValidator": {
+        "FirstLedgerSequence": 1609728,
+        "PublicKey": "ED6629D456285AE3613B285F65BBFF168D695BA3921F309949AFCD2CA7AFEC16FE"
+      }
+    }
+  ],
+  "Flags": 0,
+  "LedgerEntryType": "NegativeUNL",
+  "index": "2E8A59AA9D3B5B186B0B9E0F62E6C02587CA74A4D778938E957B6357D364B244"
+}
+```
+
+
+A `NegativeUNL` object has the following fields:
+
+| Name                  | JSON Type | [Internal Type][] | Description          |
+|:----------------------|:----------|:------------------|:---------------------|
+| `LedgerEntryType`     | String    | UInt16            | The value `0x004E`, mapped to the string `NegativeUNL`, indicates that this object is the Negative UNL. |
+| `DisabledValidators`  | Array     | Array             | _(May be omitted)_ A list of `DisabledValidator` objects (see below), each representing a trusted validator that is currently disabled. |
+| `ValidatorToDisable`  | String    | Blob              | _(May be omitted)_ The public key of a trusted validator that is scheduled to be disabled in the next flag ledger. |
+| `ValidatorToReEnable` | String    | Blob              | _(May be omitted)_ The public key of a trusted validator in the Negative UNL that is scheduled to be re-enabled in the next flag ledger. |
+
+## DisabledValidator Objects
+
+Each DisabledValidator Object represents one disabled validator. In JSON, a DisabledValidator object has one field, `DisabledValidator`, which in turn contains another object with the following fields:
```

Comment:
Should DisabledValidator be formatted in DisabledValidator Object? When you introduce other objects, you format the name of the object. 

Also, should Object be capitalized? You don't capitalize object when you introduce the the NegativeUNL object.